realm/realm-library/src/main/java/io/realm/BaseRealm.java

```diff
@@ -555,7 +555,7 @@ public RealmSchema getSchema() {
     /**
      * Deletes the Realm file defined by the given configuration.
      */
-    protected static boolean deleteRealm(final RealmConfiguration configuration) {
+    static boolean deleteRealm(final RealmConfiguration configuration) {
```

Comment:
`no modifier` is more strict i think? Even user extend the class, it still cannot be called.